src/disnake/ext/plugins/plugin.py

```diff
@@ -94,13 +95,68 @@ class SlashCommandParams(AppCommandParams, total=False):
 
 @dataclasses.dataclass
 class PluginMetadata:
+    """Represents metadata for a :class:`Plugin`
```

Comment:
Very minor nitpick-- missing period at the end of the sentence here.

Reply:
451310ccbc09b1dcf20091cef3268523f47f6e51